chapters/synchronization.txt

```diff
@@ -4773,6 +4785,18 @@ ifdef::VK_NV_shading_rate_image[]
     have been created with ename:VK_IMAGE_USAGE_SHADING_RATE_IMAGE_BIT_NV
     set
 endif::VK_NV_shading_rate_image[]
+  * If pname:image was created with a sharing mode of
+    ename:VK_SHARING_MODE_EXCLUSIVE, and pname:srcQueueFamilyIndex and
+    pname:dstQueueFamilyIndex don't describe a
+    <<synchronization-queue-transfers-acquire, queue family acquire
```

Comment:
Per styleguide, should not use contractions. "don't"

The link actually does not specify how to recognize an acquire barrier, so might be useless here. Might be better if the VU is more prescriptive. E.g.: 

> - If `image` was not created with a sharing mode of `VK_SHARING_MODE_EXCLUSIVE` then `srcAccessMask` **must** be a valid combination of `VkAccessFlagBits` values
> -  If `image` was created with a sharing mode of `VK_SHARING_MODE_EXCLUSIVE` and `dstQueueFamilyIndex` is the same as the family of the queue that will execute this barrier, then `srcAccessMask` **must** be a valid combination of `VkAccessFlagBits` values

Which technically is a problem, because we are invoking a state that cannot be validated in this spot.

The ignored parameter also impacts this `vkCmdPipelineBarrier` VU, which has not been touched:

> Each element of pMemoryBarriers, pBufferMemoryBarriers and pImageMemoryBarriers must not have any access flag included in its srcAccessMask member if that bit is not supported by any of the pipeline stages in srcStageMask, as specified in the table of supported access types.

Reply:
> Which technically is a problem, because we are invoking a state that cannot be validated in this spot.

Huh that's true, should've written it out as it would've been more obvious, I guess. Not a huge deal though, I can fix that.

Reply:
> The link actually does not specify how to recognize an acquire barrier, so might be useless here.

I've just re-read that link - why don't you think it specifies how to recognise an acquirebarrier? It says quite clearly:

> An acquire operation is defined by executing a buffer memory barrier (for a buffer range) or an image memory barrier (for an image subresource range), on a queue from the destination queue family. The buffer range or image subresource range specified in an acquire operation must match exactly that of a previous release operation. The srcQueueFamilyIndex parameter of the barrier must be set to the source queue family index, and the dstQueueFamilyIndex parameter to the destination queue family index.

What is missing from that description?

Reply:
And done. Other than the fact that I'm still referencing the link that you seemed to disagree was sufficient. Also note that vkCmdWaitEvents validation doesn't consider queue ownership transfers, which is deliberate, as (spoiler alert!) this is not actually intended to be supported and is about to be clarified in one of the next spec updates.

Reply:
> What is missing from that description?

The fact that description is not a description. It is a requirement. It says what acquire barrier is required to have as arguments. It does not say which arguments say it **is** an acquire barrier

Actual description\identification is specified elsewhere (below `VkImageMemoryBarrier`):

> If `dstQueueFamilyIndex` is not equal to `srcQueueFamilyIndex`, and `dstQueueFamilyIndex` is equal to the current queue family, then the memory barrier defines a _queue family acquire operation_ for the specified image subresource range, [...]

Reply:
Ok I see the complaint now, for some reason I thought EXTERNAL and FOREIGN had additional complications but... looks like they're basically fine 🤷‍♂ 

I was mostly just wary of duplicating info, but a simple "src != dst and {src|dst} == this queue" is fine.